com.unity.multiplayer.mlapi/Editor/ProfilerModuleOverride/NetworkProfilerOverride.cs

```diff
@@ -0,0 +1,249 @@
+using System;
+using System.Collections.Generic;
+using MLAPI.Profiling;
+using Unity.Profiling;
+using UnityEditor;
+using UnityEngine;
+
+namespace ProfilerModuleOverride
+{
+    [InitializeOnLoad]
+    public class NetworkProfilerOverride
+    {
+#if UNITY_2020_2_OR_NEWER
+        const string RPCModuleName = "MLAPI RPCs";
+        const string OperationModuleName = "MLAPI Operations";
+        const string MessageModuleName = "MLAPI Messages";
+
+        [System.Serializable]
```

Comment:
nit, don't need System.* since you already have the System namespace included

Reply:
Done